src/documentation_builder/bin/quattor-documentation-builder

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python2
+"""
+Documentation-builder generates markdown documentation.
+
+It get this from:
+ - configuration-modules-core perl documentation
+ - configuration-modules-grid perl documentation
+ - CAF perl documentation
+ - CCM perl documentation
+ - schema pan annotations
+ and creates a index for the website on http://quattor.org.
+@author: Wouter Depypere (Ghent University)
+"""
+
+import sys
+import os
+
+from vsc.utils import fancylogger
+from vsc.utils.generaloption import simple_option
+from quattordocbuild.builder import build_documentation
+
+logger = fancylogger.getLogger()
+
+REPOMAP = {
```

Comment:
hmmmm. either move this to a config file (ini-style) or better yet, add a `.docbuilder.cfg` in each repo with whatever data you need. then you only need to a list of repos here (or query all repos like this PR in release helper https://github.com/quattor/release-helper/pull/1).